speechbrain/augment/time_domain.py

```diff
@@ -493,7 +494,7 @@ def __init__(self, orig_freq, speeds=[90, 100, 110], device="cpu"):
         for speed in self.speeds:
             config = {
                 "orig_freq": self.orig_freq,
-                "new_freq": self.orig_freq * speed // 100,
+                "new_freq": int(self.orig_freq / (speed / 100)),
```

Comment:
Should this be the following to prevent rounding errors?

```suggestion
                "new_freq": self.orig_freq * 100 // speed
```

I tried with orig_freq=44100 and speed=112 and got this:
`orig_freq / (speed / 100) = 39374.99999999999`
The `int` conversion makes this `39374` which is wrong.

Reply:
On the other hand, perhaps truncating is the wrong way to go here, we could round instead:
```suggestion
                "new_freq": round(self.orig_freq * 100 / speed)
```